src/library_html5.js

```diff
@@ -971,10 +986,6 @@ var LibraryHTML5 = {
       if ({{{ makeDynCall('iipp', 'callbackfunc') }}}(eventTypeId, orientationChangeEvent, userData)) e.preventDefault();
     };
 
-    if (eventTypeId == {{{ cDefs.EMSCRIPTEN_EVENT_ORIENTATIONCHANGE }}} && screen.mozOrientation !== undefined) {
-      eventTypeString = "mozorientationchange";
-    }
```

Comment:
I assume this is some legacy firefox support.   I don't think you mentioned this change in the PR description.  I assume its really really old at this point.  Older than the oldest version of FF that we support in emscripten?   (admittedly that is not yet well defined)

Reply:
Odd, I thought I'd added a note for that. This seems to have only been supported in FF Android, and the standard event listener was supported since 2015.